chore: more pbsbatch_args usage (#5458)
determined-ai/determined: docs/reference/reference-training/experiment-config-reference.rst

            - -p1000
            - -PMyProjectName

>    Requesting of resources and job placement may be influenced through use of ``-l``, however chunk

Reviewer: I think that an example showing which "-l" options we would ignore would be helpful to the reader.

Author: Example added.

            - -p1000
            - -PMyProjectName

   Requesting of resources and job placement may be influenced through use of ``-l``, however chunk
   count, chunk arrangement, and GPU or CPU counts per chunk (depending on the value of
   ``slot_type``) are controlled by Determined; any values specified for these quantities will be

>    ignored. For example, if the following were specified for a CUDA experiment:

Reviewer 2: ```suggestion
   ignored. Consider if the following were specified for a CUDA experiment:
```

   .. code:: yaml

      pbs:
         pbsbatch_args:
            - -l select=2:ngpus=4:mem=4gb
            - -l place=scatter:shared
            - -l walltime=1:00:00

>    then the chunk count (two), the GPU count per chunk (four), and the chunk arrangement (scatter)
>    would all be ignored in favor of values calculated by Determined.

Reviewer 2: ```suggestion
   The chunk count (two), the GPU count per chunk (four), and the chunk arrangement (scatter)
   will all be ignored in favor of values calculated by Determined.
```
